### DISCUSSION TITLE
PBD particles particles spring

### DISCUSSION THREAD

**Discussion by corredD** [NVIDIA-Omniverse/PhysX#197] [Q&A]
Hello,

I'm currently in the process of transitioning my project from Flex to Physx5, specifically the PBD particle system. In Flex, particle attachment was straightforward using particle IDs and springs (NvFlexSetSprings). While working with Physx5, I've successfully used the cloth buffer to create simple chains , and it performed really well. However, when it comes to attaching chains to rigid bodies from PxParticleRigidBuffer or attaching two cloth buffer object, I dont see a way to do it. Is that something possible or plan in the futur?

Thank you!
ludovic

**Comment by sschirm** [comment]
Hi @corredD , 
We will actually deprecate particle cloth and particle rigids, as soon as we have replacements ready. Particle rigids can be replaced right now by propper PhysX rigid bodies. Particle attachments also work between particle cloth and rigid bodies.

Cheers, 
Simon
